tests/Dfe.PlanTech.Web.E2ETests/cypress/contentful-data-loader.js

```diff
@@ -0,0 +1,64 @@
+const fs = require("fs");
+
+const ContentfulDataPath = "./cypress/fixtures/contentful-data.json";
+
+let exportProcessor;
+
+/**
+ * Exports data from Contentful and writes to files for use by the dynamic-page-validator.cy.js tests
+ * @param {*} config 
+ */
+async function loadAndSaveContentfulData(config) {
+  console.log("Exporting/Loading Contentful data");
+  try {
+    exportProcessor = await import('export-processor');
+
+    await fetchContentfulExport({ exportContentfulData: exportProcessor.ExportContentfulData, config });
+  }
+  catch (e) {
+    console.error("Error loading Contentful data", e);
+  }
+}
+
+async function fetchContentfulExport({ exportContentfulData, config }) {
+  if (fs.existsSync(ContentfulDataPath)) {
```

Comment:
fileStore?

Reply:
node file system, would probably make less sense to rename because it's such a common package in JS so refactored to just leave the method names